yt_dlp/extractor/pandatv.py

```diff
@@ -0,0 +1,68 @@
+from .common import InfoExtractor
+from ..utils import (
+    ExtractorError,
+    UserNotLive,
+    int_or_none,
+    traverse_obj,
+    unified_strdate,
+    url_or_none,
+    urlencode_postdata,
+)
+
+
+class PandatvLiveIE(InfoExtractor):
+    _VALID_URL = r'(?P<base_url>https?://(?:www\.|m\.)?pandalive\.co\.kr)/play/(?P<id>[\da-z]+)'
```

Comment:
- is the site called "PandaTV"? or "PandaLive"? Calling the extractor "PandatvLive"  seems confusing

- Since we never actually request the webpage, matching the origin from the input URL is pointless, we can just hardcode `https://www.pandalive.co.kr` below

- Are you sure there's never a `-` or `.` etc in the `channel_id` ?

```suggestion
class PandatvLiveIE(InfoExtractor):
    _VALID_URL = r'https?://(?:www\.|m\.)?pandalive\.co\.kr/play/(?P<id>[\da-z]+)'
```